reference/docs-conceptual/whats-new/What-s-New-in-the-PowerShell-50-ISE.md

```diff
@@ -6,9 +6,18 @@ title:  What's New in the PowerShell 5.0 ISE
 
 # What's New in the Windows PowerShell 5.0 ISE
 
-This topic explains the new and updated features that have been introduced in versions of Windows
+This topic explains the new and updated features that have been introduced in version 5.0 of Windows

```

Comment:
change to: version 5.0 of the Windows...

Reply:
fixed